### PR TITLE
fix upload bug

### PR DESCRIPTION
**Pull request by dickeyxxx**

### CODE REVIEW

**Comment by dickeyxxx** [comment]
@ransombriggs

File: lib/storage/s3.js [hunk 1]
```diff
@@ -22,12 +22,14 @@ class S3 extends require('./base') {
   }
 
   put (key, data) {
-    let params = this._params(key, {Body: data})
-    if (data instanceof Stream) {
```
Comment (dickeyxxx) [inline lib/storage/s3.js]: this was the issue. I wasn't letting Buffer's not be json.stringified

Comment (dickeyxxx) [inline lib/storage/s3.js]: also, I'm going to make CI use S3 instead of the filesystem which doesn't test this code

**Comment by ransombriggs** [comment]
@dickeyxxx LGTM

**Pull request merged**